qt/applications/workbench/workbench/plugins/editor.py

```diff
@@ -53,8 +53,9 @@ def __init__(self, parent):
         # attributes
         self.run_action = create_action(self, "Run",
                                         on_triggered=self.editors.execute_current,
-                                        shortcut="Ctrl+Return",
+                                        shortcut=["Ctrl+Return", "Ctrl+Enter"],
```

Comment:
Maybe consider using a `tuple` for fixed-length items such as this?

Reply:
Good suggestion, I like it as it enforces it to be `const` inside the function too